src/main/java/core/basesyntax/PurchaseOperation.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Map;
+
+public class PurchaseOperation implements OperationHandler {
+    private final Map<String, Integer> fruitStorage;
+
+    public PurchaseOperation(Map<String, Integer> fruitStorage) {
+        this.fruitStorage = fruitStorage;
+    }
+
+    @Override
```

Comment:
The class `PurchaseOperation` should be placed in the `strategy` package as per the checklist to maintain a clean package structure.